tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -149,6 +150,13 @@ def keyboard(self):
         from gaiatest.apps.keyboard.app import Keyboard
         return Keyboard(self.marionette)
 
+    @property
+    def manifest_url(self):
+        return '{}{}.{}/manifest.webapp'.format(self.DEFAULT_PROTOCOL, format(self.__class__).lower(), self.DEFAULT_APP_HOSTNAME)
```

Comment:
It doesn't seem to work. 
While trying to launch the settings app, for manifest_url you get: app://<class 'gaiatest.apps.settings.app.settings'>.gaiamobile.org/manifest.webapp
Why not just use self.name.lower() here?